identity-iota/src/client/network.rs

```diff
@@ -2,15 +2,14 @@
 // SPDX-License-Identifier: Apache-2.0
 
 use identity_core::common::Url;
-use iota::client::builder;
 
 use crate::did::DID;
 
 lazy_static! {
-  static ref EXPLORER_MAIN: Url = Url::parse("https://explorer.iota.org/mainnet").unwrap();
+  static ref EXPLORER_MAIN: Url = Url::parse("https://explorer.iota.org/chrysalis").unwrap();
```

Comment:
We are dropping the `Comnet` variant I thought?

I don't really see a reason to keep the old networks. Definitely think it's a bit confusing to mix them both.

I think just `enum Network { Main, Test }` will do?

cc @JelleMillenaar